### PR TITLE
Update the documentation about running and building processes, according to the changes in Bonita 7.8

### PR DESCRIPTION
**Pull request by alachambre**

### CODE REVIEW

File: md/build-a-process-for-deployment.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ Exporting a configuration will export all the information and other items that y
...
 5. Click **Finish**. The business archive is created.
 
+or
+
+1. Right click on a diagram on the project explorer
+2. Select a process on the menu Build
+3. Specify the location where the .bar file will be created. The filename is determined by the process name and cannot be changed at this stage.
+4. Click **Finish**. The business archive is created.
+
+When you build a process from the project explorer, the configuration used is the configuration saved as default. You can edit it from the [configure menu]{configuring-a-process.md}
```
Comment (rbioteau) [inline md/build-a-process-for-deployment.md]: use '(' ')' instead of '{' '}'

**Review by rbioteau — CHANGES_REQUESTED** [review]

File: md/build-a-process-for-deployment.md [hunk 2]
```diff
@@ -15,6 +15,16 @@ Exporting a configuration will export all the information and other items that y
 4. Specify the location where the .bar file will be created. The filename is determined by the process name and cannot be changed at this stage.
 5. Click **Finish**. The business archive is created.
 
+or
+
+1. Right click on a diagram on the project explorer
+2. Select a process on the menu Build
```
Comment (educhastenier) [inline md/build-a-process-for-deployment.md:21]: process or pool? I believe in the studio, the terminology is pool

File: md/run-a-process-from-bonita-bpm-studio-for-testing.md [hunk 3]
```diff
@@ -8,7 +8,10 @@ process development.
 Before you can run a process, you must [configure the process](configuring-a-process.md)
 and [configure the organization](organization-management-in-bonita-bpm-studio.md).
 
-To run a process, open the process diagram and select the pool, then click **_Run_** in the Cool bar. The process in the selected pool runs. Only one pool can be run at a time.
+
+To run a process, you can :
+* Right click on the process diagram on the project explorer, and select the process to run from the **run** menu.
```
Comment (educhastenier) [inline md/run-a-process-from-bonita-bpm-studio-for-testing.md:13]: same remark

Comment (alachambre) [reply md/run-a-process-from-bonita-bpm-studio-for-testing.md:13]: Run a pool sounds weird in my opinion.

Comment (alachambre) [reply md/run-a-process-from-bonita-bpm-studio-for-testing.md:13]: https://documentation.bonitasoft.com/bonita/7.7/run-a-process-from-bonita-bpm-studio-for-testing
We already talk about running 'process from the Studio' in the documentation, so it's fine I think.

**Pull request merged**